src/scoping-shim.js

```diff
@@ -85,6 +85,9 @@ export default class ScopingShim {
     if (template._prepared) {
       return;
     }
+    if (!nativeShadow) {
```

Comment:
Add a comment that this is needed if customElements are not available.

Reply:
Ok